docs/developer-guide/adr-008-009-implementation-plan.md

```
@@ -0,0 +1,314 @@
+- [ ] **E11-5** Audit CLI/UI surfaces (if any) for references to the
+      removed Target fields and update printer columns, descriptions, and
+      help text.
+- [ ] **E11-6** Close out the ADR-009 "Open Questions" section with the
+      decisions made during implementation (namespace scope, status
+      location, render-registry authorization).
+
+---
+
+## Dependency Sketch
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)